w3-aggregation.md

```diff
@@ -127,20 +127,23 @@ type AggregateOffer struct {
 
 type AggregateOfferDetail struct {
   offer OfferCBOR
-  commitmentProof Proof
+  segment Link
 }
 
 type OfferCBOR any
-type Proof any
 
-type struct OfferDetails {
-    size Int
-    link Link
-    src [URL]
-    commitmentProof Proof
+type Piece {
+  size Int
+  link Link
+}
+
+type struct ContentPiece {
+  piece Piece
+  link Link
+  src? [URL]
```

Comment:
We have it described in the text for that invocation, more specifically
```
 Note that src field is optional and can be provided in a different part of the flow such as when deal is signed or through a previously agreed API.
```

Reply:
we decided to start by going this way to avoid the need to a new interaction for MVP. Also, considering we will have a redirect service for signed URLs, that will be usable here.